iconservice/deploy/icon_score_deployer.py

```diff
@@ -94,7 +101,7 @@ def _extract_files_gen(data: bytes):
             raise ScoreInstallExtractException(f'Error raising from extract_files_gen: {e}')
 
     @staticmethod
-    def deploy_legacy(path: str, data: bytes):
+    def deploy_legacy(path: str, data: bytes, *args):
```

Comment:
Why did you append this kw_arg?
I think it doesn't need this parameter

Reply:
Applied it.